server/index.js

```diff
@@ -5,32 +5,89 @@ const port = 3000;
 const bodyParser = require('body-parser');
 const axios = require('axios');
 const config = require('../config');
+const isaacAPI = require('./QuestionAnswerAPI');
+const louisAPI = require('./ProductOverviewAPI');
+const helenaAPI = require('./RatingsReviewsAPI');
 
-app.use(express.static('client'));
+app.use(express.static(`${__dirname} /../client/dist`));
 app.use(bodyParser.json());
 app.use(bodyParser.urlencoded({ extended: true }));
 
 // pass in string to get from that endpoint
 const getAPI = (params, cb) => {
-  axios({
+  const options = {
     method: 'GET',
-    url: `https://app-hrsei-api.herokuapp.com/api/fec2/hr-rpp/${params}`,
+    url: `https://app-hrsei-api.herokuapp.com/api/fec2/hr-rpp/${params}?count=100`,
```

Comment:
I'm assuming that the "?count=100" returns only 100 of the results from the API.

Reply:
I put 100 because if you don't put any value it only returns 5 entries. So setting it to 100 just makes sure we can see all the results.